### ISSUE TITLE
`license` acts in a non-standard way

### ISSUE THREAD

**Issue by 00001H** [brython-dev/brython#2532]
```py
Brython 3.13.0 on Netscape 5.0 (Windows)
Type "help", "copyright", "credits" or "license" for more information.
>>> license
Copyright (c) 2012, Pierre Quentel pierre.quentel@gmail.com
All rights reserved.
... SNIP ...
```
As per [the docs](https://docs.python.org/3/library/constants.html#license), this is what should happen:
```py
>>> license
Type license() to see the full license text
>>>
```

**Comment by PierreQuentel** [comment]
Thanks for all the reports @00001H !

**Comment by 00001H** [comment]
> Thanks for all the reports @00001H !

No worries :)
I came across this because our school was using CMU CS Acadamy which uses Brython, and after I spent so much time debugging the `list[bool]` bug I decided to find the engine to file the bug to.

Because I just really like (and apparently is good at) nitpicking/finding slightly non-conformant things, seeing a new language implementation (and a really good and comprehensive one!) piqued my interest. I decided to bust out all the knowledge I gained from reading the Python docs while bored and experiment with the most obscure of edge cases I knew. I was honestly surprised by the amount of things that did work!
